defaults/main.yml

```diff
@@ -19,3 +19,5 @@ certbot_gandi_domains: []
 #   domains:
 #     - "*.domain.example"
 
+# Destination directory for cert-assembler
+certbot_out_dir: changeme
```

Comment:
Please make this change backward compatible by setting the default to the k8router directory

Reply:
Defaulting now to `/etc/k8router/certs`